hal/src/nRF52840/sleep_hal.cpp

```diff
@@ -893,6 +908,13 @@ static int enterStopBasedSleep(const hal_sleep_config_t* config, hal_wakeup_sour
         __set_BASEPRI(basePri);
     }
 
+    if (usartRxdRdy) {
```

Comment:
Simply `|| networkRxRdy`?

Reply:
We need to distinguish `NRF_UARTE0` and `NRF_UARTE1` anyway.